lib/src/utils/rule_utils.dart

```diff
@@ -25,13 +27,25 @@ Issue createIssue({
       suggestion: replacement,
     );
 
-/// Returns the url of a page containing documentation associated with [ruleId]
+/// Returns a url of a page containing documentation associated with [ruleId]
 Uri documentation(String ruleId) => Uri(
       scheme: 'https',
       host: 'dart-code-checker.github.io',
       pathSegments: ['dart-code-metrics', 'rules', '$ruleId.html'],
     );
 
-/// Returns the [Severity] from map based [config] otherwise [defaultValue]
+/// Returns a [Severity] from map based [config] otherwise [defaultValue]
 Severity readSeverity(Map<String, Object?> config, Severity defaultValue) =>
     Severity.fromString(config['severity'] as String?) ?? defaultValue;
+
+/// Returns a list of excludes from the given [config]
+Iterable<String> readExcludes(Map<String, Object> config) {
```

Comment:
@incendial please write separate test for this function

Reply:
Done